src/assets/styles/header.scss

```diff
@@ -0,0 +1,279 @@
+    width: 80%;
+  }
+  button {
+    width: 20%;
+    background: $dark-red;
+    &:hover {
+      background: darken($dark-red, 5%);
+    }
+  }
+}
+
+.button-group {
+  margin-top: $offset;
+  overflow: auto;
+  button {
+    width: calc(50% - #{$offset / 2});
+    float: left;
+    &:last-child {
+      float: right;
+    }
+  }
+}
+
+.filter-events {
+  position: fixed;
+  z-index: 9999;
+  left: 0;
+  right: 0;
+  // NOTE: extreme magic number.
+  top: 82px;
+  bottom: 0;
+  background: $white;
+  text-align: left;
+}
+
+.filter-events-header {
+  background: $blue;
+  padding: .5em 1em;
+  color: $white;
+  font-weight: bold;
+  font-family: "Freight Sans CMP";
+  text-transform: uppercase;
+}
+
+.filter-events-body {
+  padding: 0 1em;
+  > * {
+    margin-top: 1em;
+  }
+}
+
+.filter-events-close {
+  display: block;
+  float: right;
+  &:hover {
+    cursor: pointer;
+  }
+}
+
+.filter-events-search {
+  width: 100%;
+  background: $dark-red;
+}
+
+.filter-events-title {
+  font-size: $text-size;
+  font-weight: bold;
+  font-family: "Freight Sans CMP";
+  text-transform: uppercase;
+  padding-bottom: .5em;
+  border-bottom: solid 2px $light-gray;
+  margin: 0 0 1em 0;
+}
+
+.filter-events-item {
+  margin: 1em 0;
+}
+
+.filter-events-checkbox {
+  position: relative;
+  height: 1em;
+  width: 1em;
+  border: solid 2px $black;
+  margin-right: 1em;
+  display: inline-block;
+  font-size: .8em;
+  margin-right: 1em;
+  font-weight: bold;
+  input {
+    visibility: hidden;
+  }
+  label {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    &:hover {
+      cursor: pointer;
+    }
+  }
+  input:checked + label {
+    background: $light-blue;
+  }
+}
+
+.filter-events-date-picker {
+  input {
+    border: none;
+    border-bottom: solid 1px $dark;
+    max-width: 33%;
+    padding-left: 0;
+    font-size: .8em;
+  }
+  strong {
+    font-size: .8em;
+    margin: 1em;
+  }
+}
+
+.button-dropdown {
+  position: relative;
+  display: inline-block;
+}
+
+.button-dropdown-button {
+  color: $black;
+  background: $white;
+  border: solid 1px $light-gray;
+  transition: background .3s;
+  &:hover {
+    background: $off-white;
+  }
+  span {
+    position: relative;
+    &:after {
+      position: absolute;
+      content: '';
+      height: $offset;
+      width: 0; 
+      height: 0; 
+      border-left: .3em solid transparent;
+      border-right: .3em solid transparent;
+      border-top: .3em solid $black;
+      left: calc(100% + .4em);
+      top: 50%;
+      transform: translateY(-50%);
+    }
+  }
+}
+
+.button-dropdown.open .button-dropdown-button span:after {
+  border-top: none;
+  border-bottom: .3em solid $black;
+}
+
+.button-dropdown-content {
+  position: absolute;
+  z-index: 9999;
+  top: calc(100% - 1px);
+  left: 0;
+  box-sizing: border-box;
+  width: 200%;
+  background: $white;
+  padding: 1em;
+  text-align: left;
+  border: solid 1px $light-gray;
+}
+
+
+@media (min-width: 1000px) {
```

Comment:
We could get a lot more granular here with more breakpoints, and 1000px is pretty arbitrary -- I arrived at it by just poking around a bit.